toolchain/lowering/lowering_context.cpp

```diff
@@ -127,15 +127,6 @@ auto LoweringContext::BuildFunctionDefinition(SemanticsFunctionId function_id)
 
 auto LoweringContext::BuildType(SemanticsNodeId node_id) -> llvm::Type* {
   switch (node_id.index) {
-    case SemanticsBuiltinKind::EmptyTupleType.AsInt():
-      // Represent empty types as empty structs.
-      // TODO: Investigate special-casing handling of these so that they can be
-      // collectively replaced with LLVM's void, particularly around function
-      // returns. LLVM doesn't allow declaring variables with a void type, so
-      // that may require significant special casing.
```

Comment:
Can you please move this TODO under TupleType, changing "of these" to specifically "of empty tuples"? The underlying interest in void remains.